viewing_party/party.py

```diff
@@ -1,23 +1,180 @@
 # ------------- WAVE 1 --------------------
-
+#adding a comment in the code to commit :)
 def create_movie(title, genre, rating):
-    pass
+    
+    user_data = {
+        "watched": [],
+        "watchlist": []
+        }

```

Comment:
Great job looking ahead but we don't need or use this `user_data` variable in our function -- if it ain't used, delete it!